datadog/agent/docker.go

```diff
@@ -1,40 +0,0 @@
-package agent
-
-import (
-	"github.com/DataDog/test-infra-definitions/command"
-	"github.com/DataDog/test-infra-definitions/common/config"
-
-	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
-	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-)
-
-const (
-	agentComposeDefinition = `version: "3.9"
-services:
-  agent:
-    image: gcr.io/datadoghq/agent:%s
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
The image name should be optional, default to latest docker hub release. We need to install images that sit out of docker hub, such as dev branches images